test/e2e/redundancy/gateway_failover.go

```diff
@@ -83,7 +88,17 @@ func testGatewayFailOverScenario(f *framework.Framework) {
 	// Ensure the new engine pod is started before we run the connectivity tests to eliminate possible timing issue where,
 	// after deleting the old pod, we actually run the connectivity test against the oold engine instance before k8s has
 	// a chance to react to stop the process/container etc.
-	newEnginePod := f.AwaitSubmarinerEnginePod(framework.ClusterA)
+
+	var newEnginePod *v1.Pod
+	for retries := 1; retries < 10; retries++ {
+		newEnginePod = f.AwaitSubmarinerEnginePod(framework.ClusterA)
+		if newEnginePod.Spec.NodeName == initialGatewayNode.Name {
+			time.Sleep(5 * time.Second)
+		} else {
+			break
+		}
+	}
+
```

Comment:
What is this extra retry loop for? `AwaitSubmarinerEnginePod` already has a retry loop - if the timeout needs to be longer in this case then we should pass it in.

Reply:
Ahh, ok, I didn't see that, we need to increase the timeout there then. good point

Reply:
This is already governed by the `OperationTimeout` - would bumping that to 120 sec suffice for this case?